src/components/FormSignup/signup-form-professional.jsx

```diff
@@ -2,12 +2,12 @@
 import { css } from "@emotion/react";
 import styled from "@emotion/styled";
 
-function SignUpFormProfessional() {
+function LoginFormProfessional() {
   return (
 		<>
-			<h1>Hola...</h1>
+			<h1>Hola...2</h1>
 		</>
 	);
 }
 
-export default SignUpFormProfessional;
\ No newline at end of file
+export default LoginFormProfessional;
```

Comment:
I think that this change is wrong because the file is called `signup-from-professional` and the component `LoginFormProfessional` already exists.